immersion/kubeflow_pipelines/walkthrough/solutions/lab-01_vertex.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>I believe we should just say "Vertex AI Training" (here and later below), I don't see us using "Vertex AI Platform", just "Vertex AI"</p><p> </p><p>nit: "BigQuery" and "REST API"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done!</p>